src/Symfony/Component/Validator/Tests/Constraints/ConditionalValidatorTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\Conditional;
+use Symfony\Component\Validator\Constraints\ConditionalValidator;
+use Symfony\Component\Validator\Constraints\NegativeOrZero;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\NotNull;
+use Symfony\Component\Validator\Constraints\PositiveOrZero;
+use Symfony\Component\Validator\ConstraintValidatorInterface;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+final class ConditionalValidatorTest extends ConstraintValidatorTestCase
+{
+    public function testConstraintsAreExecuted()
+    {
+        $constraints = [
+            new NotNull(),
+            new NotBlank(),
+        ];
+
+        $this->expectValidateValueAt(0, '', 'Foo', $constraints);
```

Comment:
You're right but I think this change is a little bit out of the context of this PR since this method is used by a lot of tests and your comment applies to all of them.